server/belga/publish/belga_newsml_1_2.py

```diff
@@ -804,6 +855,11 @@ def _format_administrative_metadata(self, newscomponent_2_level, item):
                 administrative_metadata, 'Property',
                 {'FormalName': 'Priority', 'Value': str(item['priority'])}
             )
+        if 'slugline' in item:
```

Comment:
imo it's better in general to check for `item.get('slugline')` so it also checks if it's not `None` which might break the output